plugins/modules/nxos_user.py

```diff
@@ -473,7 +473,7 @@ def main():
 
     # check if provided hashed password is infact a hash
     if module.params["hashed_password"] is not None:
-        if not re.match(r"^\$5\$......\$.*$", module.params["hashed_password"]):
+        if not re.match(r"^\$5\$........\$.*$", module.params["hashed_password"]):
```

Comment:
@vvas1lev Thank you for the PR. But I'm unsure if what you're proposing in this is valid for all NX-OS versions. For example:

```code
nexus9300# show version | in NXOS:
  NXOS: version 10.3(1) [Feature Release]
nexus9300# show running-config | in username
username admin password 5 $5$EJHAFP$eDk9h6gmq9A4.krEm//hHe.uDDlNXTLZfUaMQVWyWZ8 
 role network-admin
username cisco password 5 $5$JHNODB$9nuq8XjLNpYOrCJ9D6CdBbS31THfsLWp6Yq3LzfUZ78 
 role network-admin
```

The hash seems to be 6 characters here. IMO, the ideal solution should be something more generic like `\$5\$.+\$.*$`, which accounts for both the cases.